docs/src/Developers/App-Agent-Developers/Quick-Start.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Quick Start
+---
+
+# Quick Start
+
+The [Vincent Starter App](https://github.com/LIT-Protocol/vincent-starter-app) is a comprehensive reference implementation demonstrating how to build Vincent Apps that execute on-chain operations on behalf of Vincent users using their delegated Agent Wallets.
+
+This monorepo provides a full-stack implementation of a Dollar-Cost Averaging (DCA) Vincent App with frontend, backend API, and scheduled job processing.
```

Comment:
Maybe name "job processing" as an "autonomous agent execution"? Or something like that, more explicit in the sense that it is that who sends txs, without user approval in that explicit moment

Reply:
```suggestion
This monorepo provides a full-stack implementation of a Dollar-Cost Averaging (DCA) Vincent App with a frontend, backend API, and an autonomous agent that executes scheduled transactions on-chain according to the schedule permitted by the Vincent User.
```